railgun/test/integration/suite_test.go

```diff
@@ -111,7 +118,13 @@ func TestMain(m *testing.M) {
 	}
 	kongAddon := kongbuilder.Build()
 	builder := environments.NewBuilder().WithAddons(metallb.New(), kongAddon)
+
+	fmt.Println("INFO: checking for reusable environment components")
 	if existingClusterName := os.Getenv("KIND_CLUSTER"); existingClusterName != "" {
```

Comment:
another style inconsistency (inline `os.Getenv` vs global `os.Getenv`)

Reply:
historical, but since I'm changing other things anyway I'll change this.

Reply:
0b12a0a596b609caf7e97222bcc46c5dc04a5c17 resolves this, but also takes it a step further and starts storing env overrides in one consistent grouping.